WordPress/Classes/ViewRelated/Me/MeViewController.m

```diff
@@ -218,6 +219,27 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     return cell;
 }
 
+- (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger)section
+{
+    WPTableViewSectionHeaderView *header = [[WPTableViewSectionHeaderView alloc] initWithFrame:CGRectMake(0, 0, CGRectGetWidth(self.view.bounds), 0)];
+    header.title = [self titleForHeaderInSection:section];
+    return header;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section
+{
+    NSString *title = [self titleForHeaderInSection:section];
+    return [WPTableViewSectionHeaderView heightForTitle:title andWidth:CGRectGetWidth(self.view.bounds)];
+}
+
+- (NSString *)titleForHeaderInSection:(NSInteger)section
+{
+    if (section == MeSectionWpCom) {
+        return NSLocalizedString(@"WORDPRESS.COM LINKED ACCOUNT", @"WordPress.com sign-in/sign-out section header title");
```

Comment:
I should have just used the lowercase version since we are already converting it to uppercase in `WPTableViewSectionHeaderView` using `uppercaseStringWithLocale:`.